.github/workflows/build.yaml

```diff
@@ -35,6 +35,14 @@ jobs:
     - name: Set up GOPATH
       run: go env -w GOPATH=${{ github.workspace }}/go
 
+
+      - name: Check and get dependencies
```

Comment:
Offset seems wrong

Reply:
Thanks - Fixed. was hoping to iterate on it on my fork first. Don't have admin access but I think there is a toggle to allow forks to run the workflow.